engine/inc/loaders/texture/builder/texture_builder_data.hpp

```diff
@@ -24,14 +24,14 @@ class TextureBuilderData {
   std::string name;
 
   int width, height;
-  u8* data;
+  unsigned char* data;
   TextureBpp bpp;

```

Comment:
So it's here...

Reply:
Well it was really my mistake

Reply:
I know that, I was thinking changing that for u8 due to it wasn't ready the multiplataform support from that time but seeing back then i should kept lol.